testplan/web_ui/testing/src/Common/utils.js

```diff
@@ -27,6 +27,21 @@ function calcExecutionTime(entry) {
   return elapsed;
 }
 
+/**
+ * Calculate elapsed time of a timer field
+ */
+function calcElapsedTime(timerField) {
+  let elapsed = null;
+  if (timerField && Array.isArray(timerField) && !_.isEmpty(timerField)) {
+    elapsed = 0;
+    timerField.forEach((interval) => {
+      elapsed +=
+      timeToTimestamp(interval.end) - timeToTimestamp(interval.start);
+    });
+  }
+  return elapsed < 0 ? null : elapsed;
+}
+
```

Comment:
under interactive mode, if we start/stop multitest multiple times, do we get multiple timer entries (or maybe we wipe out the timer when we restart)?

Reply:
during my testing, it always had one element